### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by Nd-sole** [Ascend-Research/HeadPoseEstimation-WHENet#9]
Hey, 

Can you share the loss function implementation which you use for your paper? I want to reimplement your paper.

**Comment by lucaskyle** [comment]
i think u may check their previous works
https://github.com/haofanwang/accurate-head-pose


in the paper they mentioned:
  
 A new wrapped-loss stabilizes the network at large yaws.

Yaw prediction is divided into 120 3◦ bins covering the full range of yaws (−180◦,180◦]. 
Pitch and roll predictions are each made from 66 3◦ bins covering the range [−99◦,99◦], although
only the bins from [−90,90] are ultimately used. 

i think they just quite the same.

**Comment by Algabri** [comment]
@Nd-sole did you find their loss function? I would like to implement it with Pytorch. I did not find it here https://pytorch.org/docs/stable/nn.html#loss-functions
